docs/designers-developers/developers/richtext.md

```diff
@@ -0,0 +1,119 @@
+# RichText Reference
+
+RichText is a component that allows developers to render a [`contenteditable` input](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content), providing users with the option to format block content to make it bold, italics, linked, or use other formatting.
+
+The RichText component is extremely powerful because it provides built-in functionality you won't find in other components:
+
+* **Consistent Styling in the Admin and Frontend:** The editable container can be set to any block-level element, such as a `div`, `h2` or `p` tag. This allows the styles you apply in style.css to more easily apply on the frontend and admin, without having to rewrite them in editor.css.
+* **Cohesive Placeholder Text:** Before the user writes their content, it's easy to include placeholder text that's already styled to match the rest of the block editor.
+* **Control Over Formatting Options:** It's possible to dictate exactly which formatting options you want to allow for the RichText field. For example, you can dictate whether to allow the user to make text bold, italics or both.
+
+Unlike other components that exist in the [Component Reference](/packages/components/README.md) section, RichText lives separately because it only makes sense within the block editor, and not within other areas of WordPress.
```

Comment:
Since this will be published on WordPress.org, we should link to the handbook page here instead of linking to GitHub.

```suggestion
Unlike other components that exist in the [Component Reference](https://developer.wordpress.org/block-editor/components/) section, RichText lives separately because it only makes sense within the block editor, and not within other areas of WordPress.
```

Reply:
It’s fine as is. See our guidelines:
https://github.com/WordPress/gutenberg/blob/master/docs/contributors/document.md#using-links

We should never optimize for the handbook. It has to work on GitHub as well. There might be at least a few weeks delay with updates on WordPress.org when we are waiting for the next major release.

Reply:
This link is fine.

If in doubt, to know whether a link will work in the handbook you can 1) remove the `README.md` part and 2) add it to `https://developer.wordpress.org/block-editor/` base URL. For example, in this case, navigate to https://developer.wordpress.org/block-editor//packages/components/ and notice how it's redirected to the proper URL.

Reply:
Thanks for all the details. Just to confirm, it sounds like we should leave this one as a relative link without the suggested change. Is that correct?

Reply:
Correct :)